src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -181,7 +183,11 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
   })
 
   const getWithdrawGasEstimateCryptoBaseUnit = useCallback(
-    async (withdraw: WithdrawValues) => {
+    async (
+      withdraw: WithdrawValues,
+      quote: ThorchainSaversWithdrawQuoteResponseSuccess,
+      dustAmountCryptoBaseUnit: string,
```

Comment:
Dependency injecting vs. reacting on state values to make sure this guy gets called with non-null values, which wasn't the case previously